packages/gatsby-plugin-sitemap/README.md

```diff
@@ -23,7 +23,7 @@ generated sitemap will include all of your site's pages, except the ones you exc
 
 ## Options
 
-The `defaultOptions` [here](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-sitemap/src/internals.js#L34) can be overridden.
+The `defaultOptions` [here](https://github.com/gatsbyjs/gatsby/blob/0c39a4506493fa0409bca30c54c86daa9715a1e9/packages/gatsby-plugin-sitemap/src/internals.js#L45) can be overridden.
```

Comment:
While the line change makes sense, wouldn't it be better to link to `master` rather than sticking to a particular commit hash?

Reply:
@sidharthachatterjee Thanks for the review, I fixed it 🙂  Could you review this again?